bindings/python/README.md

```diff
@@ -1,12 +1,12 @@
 # Apache OpenDAL™ Python Binding
 
-[![](https://img.shields.io/badge/status-released-blue)](https://pypi.org/project/opendal/)
+[![img](https://img.shields.io/badge/status-released-blue)](https://pypi.org/project/opendal/)
 [![PyPI](https://img.shields.io/pypi/v/opendal.svg?logo=PyPI)](https://pypi.org/project/opendal/)
 [![Website](https://img.shields.io/badge/opendal-OpenDAL_Website-red?logo=Apache&logoColor=red)](https://opendal.apache.org/docs/python/)
 
 This package intends to build a native python binding for Apache OpenDAL.
 
-![](https://github.com/apache/opendal/assets/5351546/87bbf6e5-f19e-449a-b368-3e283016c887)
+![img](https://github.com/apache/opendal/assets/5351546/87bbf6e5-f19e-449a-b368-3e283016c887)
```

Comment:
Hi, @yihong0618, please don't change this. We have special handling for `![](xxx)` in our docs to help generate static images.

![image](https://github.com/user-attachments/assets/0e6e2b63-4a61-40fd-9b76-1b06915c2e05)

Reply:
Sorry will roll back